addOns/pscanrulesBeta/src/main/javahelp/org/zaproxy/zap/extension/pscanrulesBeta/resources/help/contents/pscanbeta.html

```diff
@@ -10,25 +10,6 @@
 <H1>Passive Scan Rules - Beta</H1>
 The following beta quality passive scan rules are included in this add-on:
 
-<H2>Cookie - Loosely Scoped</H2>
```

Comment:
Do we want to keep the headings but just say that they've been promoted to Release or will that just make it look messy?

Reply:
I think that will just end up being clutter'ish down the line, especially with some items being promoted twice. It'll be in the release notes (change log) and git history.

@thc202 what do you think?

Reply:
I think it's fine as is